docs/_sass/_main.scss

```diff
@@ -182,6 +183,195 @@ button {
     align-content: center;
 }
 
+.logo {
+    width: 80%;
+}
+
+.search-icon {
+    margin: auto 0px;
+}
+
+.flex {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -moz-flex;
+    display: -webkit-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;
+    flex-flow: row wrap;
+    align-content: space-between;
+}
+
+#sidebar-search {
+    background-color: $color-green-appBG;
+    width: 375px;
+    height: 100vh;
+    position: fixed;
+    display: block;
+    top: 0;
+    right: 0;
+}
+
+@media only screen and (max-width: $breakpoint-tablet) {
+    #sidebar-search {
+        width: 100%;
+    }
+}
+
+.searchbar-title-wrapper {
+    padding: 20px;
+}
+
+.search-title {
+    font-size: 17px;
+    padding-bottom: 20px;
+}
+
+#toggle-search-close {
+    margin: auto;
+    margin-left: 0px;
+    margin-right: 10px;
+}
+
+/* Sidebar Layer */
+#sidebar-layer {
+    position: fixed;
+    /* Sit on top of the page content */
+    display: none;
+    /* Hidden by default */
+    width: 100%;
+    /* Full width (cover the whole page) */
+    height: 100%;
+    /* Full height (cover the whole page) */
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.4);
+    z-index: 2;
+    /* Specify a stack order in case you're using a different order for other elements */
+}
+
+/* All gsc id & class are Google Search related */
+#___gcse_0 {
+    margin-left: 20px;
+}
+
+input#gsc-i-id1.gsc-input {
+    background-color: $color-green-appBG;
+    color: #E7ECE9;
+    font-family: "ExpensifyNeue", "Segoe UI Emoji", "Noto Color Emoji" !important;
+}
+
+#gsc-iw-id1 {
+    background-color: $color-green-appBG;
+    border-bottom: rgba(255, 255, 255, 0.2) 2px solid;
+    border-bottom-left-radius: 0px;
+    &:focus-within {
+        border-bottom: $color-accent 2px solid;
+    }
+}
+
+.gsc-input-box .gsib_a {
+    padding: 0;
+}
+
+.search-icon {
+    margin-left: auto;
+}
+
+/* Manage Google Search label animation */
+input:focus+label,
+input:valid+label,
+input:active+label {
+    transform: translateY(-100%) scale(0.80);
+}
+
+label {
```

Comment:
Should we give this more specificity? Otherwise it will impact all labels, not just those specific to google search.